packages/acisfp_check/test_regress_head.sh

```diff
@@ -0,0 +1,4 @@
+acisfp_check \
+   --outdir=headout \
+   --oflsdir=/data/acis/LoadReviews/2018/MAY2818/oflsa \
+   --run-start=2018:142
```

Comment:
The default is --state-builder = 'acis'.  This would be more obvious if I just called it out here.